.github/workflows/screenshots.yml

```diff
@@ -10,10 +10,6 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-permissions:
-  contents: read
```

Comment:
This should stay to restrict permissions for GitHub tokens, 5f760025004bdb02f9844011033459c30347f215.

https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/

Reply:
👍🏻, done